src/docs/product/accounts/sso/okta-sso.mdx

```diff
@@ -11,49 +11,93 @@ description: "Set up Okta Single Sign-On (SSO) on Sentry."
 1. From the admin console's sidebar, click **Applications > Applications**, then click on "Add Application".
 
 1. Search for `Sentry` and click "Add", and you will be brought to the setup wizard.
-   
+
    ![Sentry in Okta Gallery of Integrations](okta-gallery.png)
-   
-1. For "1. General Settings", no special settings are needed. 
 
-1. For "2. Sign On Options", fill in these fields as follows: 
-   
+1. For "1. General Settings", no special settings are needed.
+
+1. For "2. Sign On Options", provide the following:
+
    - Base URL: `https://sentry.io`
-      
+
       - Important: There should not be a trailing slash at the end of the URL as it will break the redirect from Okta to Sentry.
-      
+
    - Organization Slug: *YOUR_ORG_SLUG*
-   
+
       - You can find your organization slug in the URL: `https://sentry.io/organizations/YOUR_ORG_SLUG/`
-      
+
    - Email: `Email`
```

Comment:
```suggestion
   - Email: your email ID
```

Reply:
i think this one should stay as ``Email``, as that is the name of the specific option from the dropdown you'd select in okta.